### DISCUSSION TITLE
Slack integration with tags filtering

### DISCUSSION THREAD

**Discussion by theSculpt0r** [cypress-io/cypress#30364] [Cypress Cloud]
Hi! Could you please explain me how to configure filtering by tags? Field tags is not editable. Current plan is "team".
![image](https://github.com/user-attachments/assets/371c50d7-466a-4522-968e-2161973fe75f)

**Comment by eswanny22** [comment]
Please make sure the Slack integration is installed at the Project level instead of the Organization level, this will allow the tags option to be clickable.
